origins_translations/src/Plugin/Block/OriginsTranslationBlock.php

```diff
@@ -82,7 +98,84 @@ public static function create(ContainerInterface $container, array $configuratio
    * {@inheritdoc}
    */
   public function build() {
-    return $this->formBuilder->getForm('Drupal\origins_translations\Form\LanguageSelectorForm');
+    $request = $this->request;
+    $languages = $this->utilities->getActiveLanguages();
+
+    // Determine the URL to be translated.
+    $url = $request->getBaseUrl();
+
+    // Create links to Google Translate.
+    $translation_links = [];
+    foreach ($languages as $code => $language) {
+      $link_text = Markup::create($language[0] . ' &mdash; <span lang="' . $code . '" dir="' . $language[5] . '">' . $language[1] . '</span>');
```

Comment:
A small nit-pick around references such as `$language[5]` - it's tricky to see what these are if looking at the code for the first time or at a later date. I'd consider some kind of named reference variable... something like `$lang_text = &$language[1];` for example, so that when you're passing it as parameters into something else it's a lot clearer what each variable is for.